src/models/User.model.js

```diff
@@ -1,9 +1,27 @@
 'use strict';
 
+const { DataTypes } = require('sequelize');
 const { sequelize } = require('../db.js');
 
 const User = sequelize.define(
-  // your code goes here
+  'User',
+  {
+    id: {
+      type: DataTypes.INTEGER,
+      autoIncrement: true,
+      allowNull: false,
+      primaryKey: true,
+    },
+    name: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+  },
+  {
+    tableName: 'users',
+    createdAt: false,
+    updatedAt: false,
```

Comment:
The `createdAt` and `updatedAt` fields are set to `false`, which means Sequelize will not automatically manage these timestamps. If you need to track when user records are created or updated, consider setting these to `true` or adding custom timestamp fields.